index.js

```diff
@@ -29,4 +48,12 @@ module.exports = function run() {
     })
     .then(() => spinner.succeed('Successfully dropped and created database!'))
     .catch(error => spinner.fail(error.message));
+}
+
+module.exports = () => {
+  if (process.argv.includes('init')) {
```

Comment:
I would consider using yargs (https://www.npmjs.com/package/yargs) for handling arguments